graphene/src/matrix.rs

```diff
@@ -200,4 +201,41 @@ impl Matrix {
             out.assume_init()
         }
     }
+
+    pub fn values(&self) -> [[f32; 4]; 4] {
```

Comment:
I think technically you could also return a `&[[f32; 4]; 4]` here.

```rust
&*(&self.value as *const ffi::graphene_simd4x4f_t as *const [[f32; 4]; 4])
```

But that should come with a test please. I think that would be useful to do though as otherwise you'd always copy 16 floats here on every access, which is a bit :)